VS/VehicleSignals.rdf

```diff
@@ -2954,6 +2956,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;LeftIndicatorLight">
 		<rdfs:subClassOf rdf:resource="&auto-vs;ActuableSignal"/>
+		<rdfs:label xml:lang="en">Left indicator light</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.